imports/plugins/core/accounts/client/templates/profile/profile.js

```diff
@@ -67,54 +66,32 @@ Template.accountProfile.helpers({
   },
 
   /**
-   * User's account profile
-   * @return {Object} account profile
-   */
-  account() {
-    return Collections.Accounts.findOne();
-  },
-
-  /**
    * User's display name
    * @return {String} display name
    */
   displayName() {
-    const userId = Meteor.userId() || {};
-    const user = Collections.Accounts.findOne(userId);
-
-    if (user) {
-      if (user.name) {
-        return user.name;
-      } else if (user.username) {
-        return user.username;
-      } else if (user.profile && user.profile.name) {
-        return user.profile.name;
+    const account = Collections.Accounts.findOne(Meteor.userId());
```

Comment:
We shouldn't need to subscribe here because we've got a global subscription for this, but it's probably worth making sure the subscription is ready.

https://github.com/reactioncommerce/reaction/blob/spencer-2889-cleanup-marketplace-shop-creation/client/modules/core/subscriptions.js#L15